extensions/sinks/redis/redis.go

```diff
@@ -0,0 +1,209 @@
+package main
```

Comment:
Please add copyright statement at the top. Check other source file for reference, and sign with your name or email.

Reply:
ok